qutip/solver/stochastic.py

```diff
@@ -706,7 +707,7 @@ class SMESolver(StochasticSolver):
         "store_measurement": False,
         "keep_runs_results": False,
         "normalize_output": False,
-        "method": "taylor1.5",
+        "method": "platen",
```

Comment:
Just checking that the change to the default method here was intended.

Reply:
Yes, I matched the method with `ssesolve` and the documentation default. We had 3 different values.